bouncerscript/constants.py

```diff
@@ -29,6 +29,15 @@
     'firefox-sha1-ssl': r'^Firefox-\d+\.\d+(\.\d+)?esr-sha1$',
 }
 
+PARTNER_ALIASES_REGEX = {
+    r'partner-firefox-beta-(.*)-latest': r'^Firefox-\d+\.0b\d+-(.*)$',
```

Comment:
do we need `^` and `$` at the beginning and end of the alias pattern to avoid potentially matching strings where this is a substring?

Reply:
That's a good idea, I'll add it.